### PR TITLE
Refactor get isotope

### PR DESCRIPTION
**Pull request by shubhra-agrawal**
Abundance error and correlation check are done both in isotopeDetection and isotopeWidget. Moving the code block into a separate function to get rid of duplicate code later.

### CODE REVIEW

File: src/core/libmaven/isotopeDetection.h [hunk 1]
```diff
@@ -26,6 +26,7 @@ class IsotopeDetection
 		bool D2Flag);
 
 	void pullIsotopes(PeakGroup *group);
+	bool filterIsotope(Isotope x, bool C13Flag, bool N15Flag, bool S34Flag, bool D2Flag, float parentPeakIntensity, float isotopePeakIntensity, mzSample* sample, PeakGroup* parentGroup = NULL);
```
Comment (IpankajI) [inline src/core/libmaven/isotopeDetection.h]: break this line with no. of character not more than 80

Comment (shubhra-agrawal) [reply src/core/libmaven/isotopeDetection.h]: @IpankajI Changed in future branches. Most of the parameters have been removed as well.

File: src/core/libmaven/isotopeDetection.cpp [hunk 2]
```diff
@@ -202,6 +164,58 @@ map<string, PeakGroup> IsotopeDetection::getIsotopes(PeakGroup* parentgroup, vec
     return isotopes;
 }
 
+bool IsotopeDetection::filterIsotope(Isotope x, bool C13Flag, bool N15Flag, bool S34Flag, bool D2Flag, float parentPeakIntensity, float isotopePeakIntensity, mzSample* sample, PeakGroup *parentGroup)
```
Comment (IpankajI) [inline src/core/libmaven/isotopeDetection.cpp]: @shubhra-agrawal Break all lines with no. of character not more than 80

Comment (shubhra-agrawal) [reply src/core/libmaven/isotopeDetection.cpp]: @IpankajI Done in future branches.

**Pull request merged**